Nuget/Ploeh.AutoFixture.NUnit3.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.NUnit3</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with NUnit3</title>
+        <authors>Hackle Wayne, AutoFixture</authors>
+        <owners>AutoFixture</owners>
+        <dependencies>
+            <dependency id="autofixture" version="@build.number@" />
+            <dependency id="nunit" version="3.0.1" />
+        </dependencies>
+        <summary>Extension to AutoFixture that integrates it with NUnit3.</summary>
+        <description>By leveraging the some features of NUnit3, this extension turns AutoFixture into a declarative framework for writing unit tests. In many ways it becomes a unit testing DSL (Domain Specific Language).</description>
```

Comment:
Is there a rogue _the_ here?

Sorry to be nitpicking, but this text goes on the NuGet package page, so need to be as descriptive and trustworthy as possible.